test/Lint.lean

```diff
@@ -254,3 +254,64 @@ info: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20,
 -/
 #guard_msgs in
 #eval List.range 27
+
+/-
+# Testing the `longFile` linter
+
+Things to note:
+* `set_option linter.style.longFile 0` disables the linter, allowing us to set a value smaller than
+  `1500` without triggering the warning for setting a small value for the option;
+* `set_option ... in #exit` allows processing of the file *beyond* `#exit`, since it wraps
+  `#exit` inside an anonymous section, making Lean active again *after* that anonymous section.
+-/
+
+/--
+warning: The default value of the `longFile` linter is 1500.
+The current value of 1500 does not exceed the allowed bound.
+Please, remove the `set_option linter.style.longFile 1500`.
+-/
+#guard_msgs in
+-- Do not allow setting a "small" `longFile` linter option
+set_option linter.style.longFile 1500
+
+/--
+warning: using 'exit' to interrupt Lean
+---
+warning: The default value of the `longFile` linter is 1500.
+This file is 166 lines long which does not exceed the allowed bound.
+Please, remove the `set_option linter.style.longFile 1600`.
+-/
+#guard_msgs in
+-- Do not allow increasing unnecessarily the `longFile` linter option
```

Comment:
Not a native speaker, but is this better grammar? (If you know better, feel to ignore this/enlighten me!)
```suggestion
-- Do not allow unnecessarily increasing the `longFile` linter option